airgun/views/product.py

```diff
@@ -238,3 +238,19 @@ class ProductSyncPlanView(SyncPlanCreateView):
     def is_displayed(self):
         return self.browser.wait_for_element(
             self.title, exception=False) is not None
+
+
+class ProductManageHttpProxy(BaseLoggedInView):
+    """Represents Http Proxy Management page for Products."""
+    title = Text("//h4[text()='Assign Compliance Policy]")
```

Comment:
That won't match anything, title is "Http Proxy Management". And you have unclosed single quote.

Please take another look at `airgun/views/host.py`. Specific modal views have `title` set, because they all inherit from `HostsActionCommonDialog`, that will provide `is_displayed` that waits until title is displayed (assumes "title is displayed" means "modal is fully loaded").

I think you should add `is_displayed` here, too, or make this inherit from `ProductsActionCommonDialog` that will define `is_displayed`. Either way works for me.

Reply:
Added `is_displayed` and fixed `title`.